### ISSUE TITLE
Better error reporting.

### ISSUE THREAD

**Issue by YurySolovyov** [dependencyci/support#14]
`"An error occured during the build"` is not that useful unless you know what the error was , or can see the logs of some sort.

**Comment by mbaechler** [comment]
That the same here, see https://dependencyci.com/github/linagora/james-project/builds/1

**Comment by andrew** [comment]
Agreed, it's pretty poor right now, I'll be improving that over the next few days.

**Comment by mbaechler** [comment]
Thank you @andrew but success is not much helpful : 0 dependencies found, obviously wrong.

**Comment by danez** [comment]
I have the same problem currently here, and no clue what is wrong: https://dependencyci.com/github/chili-labs/react-fast-highlight/builds/16

**Comment by vitorbal** [comment]
We also started having the same issue, in case it helps debugging: https://dependencyci.com/github/eslint/eslint/builds/84

**Comment by andrew** [comment]
@danez @vitorbal sorry about that, had a server issue whilst I was travelling, have fixed it up now, still working on improving the error reporting without leaking the actual exception errors that have happened.

**Comment by vitorbal** [comment]
No worries! Thanks for the quick reply!

**Comment by aeweidne** [comment]
Our org is experiencing the same failures as well, here is one for example: https://dependencyci.com/github/ibm-cloud-solutions/hubot-ibmcloud-service-suggest/builds/19

Is there a status page for the service that we could check?

**Comment by andrew** [comment]
@aeweidne there's no status page at the moment but you can follow the twitter account for details on any outages: https://twitter.com/dependencyci

The database will be upgraded this weekend should solve most of the intermittent build failures.

**Comment by vrtxf** [comment]
getting unknown error for about 2 weeks as well
